client/state/data-layer/wpcom/auth/send-login-email/index.js

```diff
@@ -50,11 +52,18 @@ export const sendLoginEmail = ( action ) => {
 		...( loginFormFlow
 			? [ recordTracksEventWithClientId( 'calypso_login_block_login_form_send_magic_link' ) ]
 			: [] ),
+		...( createAccount
+			? [
+					recordTracksEventWithClientId(
+						'calypso_login_block_login_form_send_account_create_magic_link'
+					),
+			  ]
+			: [] ),
 		http(
 			{
 				path: `/auth/send-login-email`,
 				method: 'POST',
-				apiVersion: '1.2',
```

Comment:
Version 1.2 doesn't support account creation through magic links. Version 1.3 does it.